### PR TITLE
BD-2760 Revamp Custom Attributes - operators

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #BD-2760

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md [hunk 1]
```diff
@@ -103,19 +103,19 @@ String attributes are useful for storing user input, such as a favorite brand, a
...
-| Check if the string attribute **does not exactly match any** of the inputted strings | **IS NONE OF** | **STRING**<br>Case sensitive; multiple strings allowed |
-| Check if the string attribute **partially matches any** of the inputted strings | **CONTAINS ANY OF** | **STRING**<br>Case sensitive; multiple strings allowed |
-| Check if the string attribute **does not partially match any** of the inputted strings | **DOESN'T CONTAIN ANY OF** | **STRING**<br>Case sensitive; multiple strings allowed |
-{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
+| Segmentation options | Dropdown filter | Input options | Examples |
+| ---------------------| --------------- | ------------- | -------- |
+| Check if the string attribute **exactly matches** an inputted string| **EQUALS** | **STRING**<br>Case sensitive | If this filter specifies `book` and a user profile has a string attribute for `last_item_purchased` that contains `book`, the user will match this filter. |
+| Check if the string attribute **partially matches** an inputted string **OR** Regular Expression | **MATCHES REGEX** | **STRING** **OR** **REGULAR EXPRESSION** <br>Not case sensitive; maximum of 32,764 characters |
```
Comment (josh-mccrowell-braze) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md:109]: Neither this row or the one after it has an example. Should it?

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md [hunk 2]
```diff
@@ -144,20 +144,20 @@ Note that If you input any values with spaces in between, before, or after words
...
-| Check if the array attribute **does not include all of** the inputted values | **ISN'T ALL OF** | **STRING**<br>Case sensitive; multiple values allowed |
-{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3}
+| Segmentation options | Dropdown filter | Input options | Examples |
+| ---------------------| --------------- | ------------- | -------- |
+| Check if the array attribute **includes a value which exactly matches** an inputted value| **INCLUDES VALUE** | **STRING** | If this filter specifies `sci-fi` and a user profile has the value `sci-fi`, the user will match this filter.|
+| Check if the array attribute **does not include a value which exactly matches** an inputted value| **DOESN'T INCLUDE VALUE** | **STRING** | If this filter specifies `sci-fi` and a user profile doesn't have the value `sci-fi`, the user will match this filter.|
+| Check if the array attribute **contains a value which partially matches** an inputted value **OR** Regular Expression | **MATCHES REGEX** | **STRING** **OR** **REGULAR EXPRESSION**<br>Maximum of 32,764 characters | |
```
Comment (josh-mccrowell-braze) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md:151]: This row doesn't have an example. Should it?

**Review by josh-mccrowell-braze — APPROVED** [review]
I had two comments where rows seemed to be missing examples (and it felt kind of weird), but all of the examples seem to match the segmentation option. 

These examples are really doing a lot to make sorta jargony, logic-based options be gettable. Good job.

Approving to not block merge. Once you've added more examples - or decided they don't need them - this can be merged!

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md (hunk 2 above)
Comment (rachel-feinberg) [reply _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md:151]: Wendy gave the green light to leave regex operators without examples:
> there’s a lot of customization you can do with regex and those who use it are typically more technical

File: _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md (hunk 1 above)
Comment (rachel-feinberg) [reply _docs/_user_guide/data_and_analytics/custom_data/custom_attributes.md:109]: The decision for now is to leave regex blank, but I can return to that during the ticket bash. Wendy's OOO today, and will be back on Monday.

**Pull request merged**